lib/diagnostic.rb

```diff
@@ -8,21 +8,21 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'require 'lib/example.rb''
 
```

Comment:
In ruby you can run scripts in ruby with the `ruby` keyword
`ruby 'lib/example.rb'`